docs/openCypher.md

```diff
@@ -0,0 +1,54 @@
+# openCypher Data Format
+
+openCypher was chosen as it represents “the most widely adopted, fully-specified, and open query
+language for property graph databases”. Allowing customers to import data from enterprise applications
+such as neo4j and Amazon Neptune with greater ease. One caveat to this is that there are slight differences
+in the System column header formats used by each, these differences are shown below.
+#### Entity (Node) System column headers
+- :ID / _id  –   (Required) An ID for the node.
+- :LABEL / _labels  –   A label for the node.
+
+#### Edge (relationship) System column headers
+- :ID / _id   –   An ID for the relationship.
```

Comment:
Maybe also this could tell users which field maps to each Gaffer field, i.e. `:START_ID` -> `SOURCE`